Core.ApplicationServices/KLEApplicationService.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections.Generic;
+using Core.ApplicationServices.Authorization;
+using Core.ApplicationServices.Model.Result;
+using Core.DomainModel.KLE;
+using Core.DomainModel.Organization;
+using Core.DomainServices.Model.Result;
+using Core.DomainServices.Repositories.KLE;
+
+namespace Core.ApplicationServices
+{
+    public class KLEApplicationService : IKLEApplicationService
+    {
+        private readonly IOrganizationalUserContext _organizationalUserContext;
+        private readonly IKLEStandardRepository _kleStandardRepository;
+        private readonly IKLEUpdateHistoryItemRepository _kleUpdateHistoryItemRepository;
+
+        public KLEApplicationService(IOrganizationalUserContext organizationalUserContext,
+            IKLEStandardRepository kleStandardRepository,
+            IKLEUpdateHistoryItemRepository kleUpdateHistoryItemRepository)
+        {
+            _organizationalUserContext = organizationalUserContext;
+            _kleStandardRepository = kleStandardRepository;
+            _kleUpdateHistoryItemRepository = kleUpdateHistoryItemRepository;
+        }
+
+        public Result<KLEStatus, OperationFailure> GetKLEStatus()
+        {
+            if (!_organizationalUserContext.HasRole(OrganizationRole.GlobalAdmin))
```

Comment:
@strongmindsmhs  Der er vel kun 1 regel .. er der så ikke bare 1 metode: Allow --> bool?

Reply:
@mrjsawdk Jeg er ikke helt med - Der returneres en KLEStatus, som er en angivelse af om en update giver mening (bool) og datoen for det aktuelle KLE sæt (hos KLE-Online). Værdierne anvendes i UI til en meningsfuld forklaring af UI button state

Reply:
Min kommentar gå på at HasRole (GlobalAdmin) er gentaget i alle metoder. Reglen er generel hvorfor der blot bør være 1 metode "HasAccess" i KLEApplicationService .. DRY ;-)